app/main.py

```diff
@@ -2,24 +2,64 @@
 
 
 class IntegerRange:
-    pass
+    def __init__(self, min_amount: int, max_amount: int) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
+
+    def __set_name__(self, owner: any, name: str) -> None:
+        self.protected_name = "_" + name
+
+    def __get__(self, instance: any, owner: any) -> int:
+        return getattr(instance, self.protected_name)
+
+    def __set__(self, instance: any, value: int) -> None:
+        if not self.min_amount <= value <= self.max_amount:
+            raise ValueError(
+                f"The value must be in the range "
+                f"{self.min_amount} to {self.max_amount}"
+            )
+        setattr(instance, self.protected_name, value)
```

Comment:
The IntegerRange class is well implemented, but it seems to be more general than it needs to be. If it's only used for age, height, and weight validation, consider renaming it to something like 'AgeRange', 'HeightRange', or 'WeightRange' to make its purpose more clear.